File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/I. Palindrome/main.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/I
#include <bits/stdc++.h>
using namespace std;
 
int main()
{
    int x, pal=0;
    cin >> x;
    int new_x = x;
 
    while(new_x){
        pal *= 10;
        pal += new_x%10;
        new_x /=10;
    }
    cout << pal << endl;
    cout << (pal == x ? "YES" : "NO");
    return 0;
}

File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/I. Palindrome/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/I

using System;

namespace Codeforces.Assuit.Contest1
{
    class ProblemI
    {
        static void Main(string[] args)
        {
            int x, pal=0;
            x = int.Parse(Console.ReadLine());
            int new_x = x;
            while(new_x > 0){
                pal *= 10;
                pal += new_x%10;
                new_x /=10;
            }
            Console.WriteLine(pal);
            Console.WriteLine((pal == x ? "YES" : "NO"));
        }
    }
}
